src/pages/deriv-go/_banner.tsx

```diff
@@ -112,6 +110,42 @@ const DHero = () => {
     const [is_logged_in] = useAuthCheck()
     const is_rtl = useIsRtl()
 
+    const getMobileOperatingSystem = () => {
```

Comment:
Hi @aswathy-deriv I guess we have this function. 
Can you please check this file if its useful for you. `/src/common/os-detect.ts`

Reply:
yeah its already there.okay i will use it from there then.thanks @fasihali-deriv